data_architect_misc/data_transformer/configs/APAC_Taiwan/step3_combine_tranformed_noncompete_and_compete_data.json

```diff
@@ -11,6 +11,7 @@
     "write_output": true,
     "output_folder_path": "./output/APAC_Taiwan/",
     "output_file_name_prefix": "Combined_Taiwan_Investment_20200501_20200531_",
+    "output_encoding" : "utf-8-sig",
```

Comment:
Leave it as "utf-8"?

Reply:
It's no possible sire or we should find another soluction

Reply:
Please leave a comment before "output_encoding" like below:
"__output_encoding_comment__": "In Taiwan input files, we have to use utf-8-sig as output encoding or else it does not work. The character outputs become incorrect if we use utf-8."